README.md

```diff
@@ -16,6 +16,8 @@ Therefore, for nix-direnv, you probably need something like:
 echo 'use flake . --impure' > .envrc
 ```
 
+NB: if you have nvidia hardware, you'll also need `export NIXPKGS_ALLOW_UNFREE=1`
```

Comment:
Why not suggesting to add `export NIXPKGS_ALLOW_UNFREE=1` to the .envrc and provide the following example:
```bash
echo 'export NIXPKGS_ALLOW_UNFREE=1' > .envrc && echo 'use flake . --impure' >> .envrc
```

Reply:
Did you test that ? If this work this is totally fine for me

Reply:
Yes I tested this command. This is how I fixed my issues with nvidia being unfree software on my laptop.